docs/release-notes/latest.md

```diff
@@ -9,100 +9,101 @@ tags:
 
-### Added
+### Changed
 
-- **Filtering:** Allow users to select more than one branch at a time.
+- **SBOM export:** The name of the exported SBOM file now follows the following format: `sbom-<org_name>-<repo_name>-<MM-DD-YY_H-m-s>--<serial_number>.<xml|json>` <!-- (10850) -->
 
-## 🤖 Semgrep Assistant (beta)
+## 🔐 Semgrep Secrets (beta)
 
 ### Added
 
-- **GitLab:** Semgrep Assistant now supports GitLab cloud hosted and self-managed repositories.
-- **Findings page**: Semgrep Assistant verdicts now appear in the Findings page if Assistant recommends that the finding should be **Ignored**. <!-- #10438 -->
-![Sample finding entry with Semgrep Assistant verdict](/img/sept-2023-assistant-findings.png)
-- **Finding Details page:** For findings with autofixes, the finding's detail page includes a link to the PR comment with the autofix since the PR comment allows for directly committing the autofix. <!-- #10516 -->
+- Semgrep Secrets is now in **public beta**.
+- **Projects page:** Added a new column to display a Semgrep Secrets counter. This counter counts all secrets regardless of validation state. <!--(10588)-->
 
 ### Fixed
 
-- **GitLab:** Fixed a bug in which comments were not appearing on GitLab.com cloud-hosted repositories.
+- Fixed links to branches in GitLab self-hosted repositories. <!-- (#10897) -->
 
 ## 📝 Documentation and knowledge base
 
 ### Added
-
-* New knowledge base articles:
-    * [<i class="fa-regular fa-file-lines"></i> Failed to run a git command during pull or merge request scans](/kb/semgrep-ci/git-command-errors/)
-    * [<i class="fa-regular fa-file-lines"></i> How to exclude certain file types for a particular rule](kb/rules/exclude_rule_for_certain_filetypes/)
-    * [<i class="fa-regular fa-file-lines"></i> Why isn’t Semgrep reporting all my tainted data flows?](kb/semgrep-code/finding_all_taints/ )
-    * [<i class="fa-regular fa-file-lines"></i> How to scan multiple or nested lock files](kb/semgrep-supply-chain/scanning_multiple_lockfiles/)
-* [<i class="fa-regular fa-file-lines"></i> Semgrep Assistant](/semgrep-code/semgrep-assistant-code/#enabling-semgrep-assistant): Added a guide to setting up Assistant on GitLab MRs.
-* [<i class="fa-regular fa-file-lines"></i> Supported languages](/supported-languages/#language-maturity-factors-pro-engine): Added a section on Semgrep Pro Engine language maturity factors. These are the criteria that determine if a language is generally available (GA) or beta.
+* Added Semgrep Secrets documentation:
+	* [<i class="fa-regular fa-file-lines"></i> Conceptual overview of Semgrep Secrets](/semgrep-secrets/conceptual-overview)
+	* [<i class="fa-regular fa-file-lines"></i> Getting started with Semgrep Secrets](/semgrep-secrets/getting-started)
+* Added [<i class="fa-regular fa-file-lines"></i> Repository rulesets](/kb/semgrep-ci/github-repository-rulesets-semgrep/) knowledgebase article. This article explains how to scale Semgrep across many GitHub repositories.
```

Comment:
```suggestion
* Added [<i class="fa-regular fa-file-lines"></i> Repository rulesets](/kb/semgrep-ci/github-repository-rulesets-semgrep/) knowledge base article. This article explains how to scale Semgrep across many GitHub repositories.
```

Reply:
I'm pretty sure I should have written a Vale rule for this :) Thank you.